From 8cfc723a596bcdab404f4861571b178d76eb1f7a Mon Sep 17 00:00:00 2001
From: Stefan Saraev <stefan@saraev.ca>
Date: Tue, 28 Oct 2014 22:32:18 +0200
Subject: [PATCH] stop execute_preset doing stupid things

populating shitload of useless symlinks into /storage/.config/system.d
is not so-nice fuckery. and systemctl preset-all is useless on an
embedded distro like openelec anyway.
---
 src/shared/install.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/src/shared/install.c b/src/shared/install.c
index 17e03e5..f7e51af 100644
--- a/src/shared/install.c
+++ b/src/shared/install.c
@@ -2165,6 +2165,11 @@ static int execute_preset(
                 UnitFileChange **changes,
                 unsigned *n_changes) {
 
+        // meh systemctl preset-all.
+        // I DONT WANT shitload of symlink populated
+        // just because systemd "thinks" (o_O) /etc is empty
+        return 0;
+
         int r;
 
         assert(plus);
